src/WarcraftLegacies.Source/Quests/Frostwolf/QuestThunderBluff.cs

```diff
@@ -35,7 +35,7 @@ public sealed class QuestThunderBluff : QuestData
     }
 
     //todo: bad flavour
-    protected override string CompletionPopup => "The Crossroads have been constructed.";
+    protected override string CompletionPopup => "The Long March of the Tauren clans has ended and they have joined forces with the Horde.";
```

Comment:
Nice, I like the flavour a lot.
```suggestion
    protected override string CompletionPopup => "The long march of the Tauren clans has ended, and they have joined forces with the Horde.";
```